docs/scripting/functions/RedirectDownload.md

```diff
@@ -0,0 +1,64 @@
+---
+id: RedirectDownload
+title: RedirectDownload
+description: Redirect a player custom AddCharModel or AddSimpleModel download to a specific HTTP webpage.
+tags: []
+---
+
+:::note
+
+This function name starts with a lowercase letter.
```

Comment:
🤔

Reply:
This was due to me forgetting to change the note's text after copying it from another wikipage. It is resolved now.